src/components/Project/index.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react';
+
+import TagsList from 'Components/TagsList';
+import PropTypes from 'prop-types';
+
+import { minutesToHours } from 'date-fns/esm';
+
+import { ReactComponent as ArrowRigth } from 'Assets/arrow-rigth.svg';
+import { ReactComponent as ArrowDown } from 'Assets/arrow-down.svg';
+
+import './index.scss';
+
+const Project = ({
+  tags, color, name, trackedTime,
+}) => {
+  console.log(tags);
+  const [minutes, setMinutes] = useState('');
+  const [open, setOpen] = useState(false);
+
+  const opening = () => {
+    if (!open) {
+      setOpen(true);
+    } else {
+      setOpen(false);
+    }
+  };
```

Comment:
Que opinas de hacer algo de este estilo?
```suggestion
  const opening = () => setOpen(!open);
```